content/applications/sales/sales/products_prices/products/variants.rst

```diff
@@ -122,12 +101,41 @@ once an attribute is added to a product.
 .. warning::
    Once added to a product, an attribute's :guilabel:`Variants Creation Mode` cannot be edited.
 
-Lastly, the :guilabel:`eCommerce Filter Visibility` field determines whether these attribute options
+The :guilabel:`eCommerce Filter Visibility` field determines whether or not these attribute options
 are visible to the customer on the front-end, as they shop on the online store.
 
 - :guilabel:`Visible`: the attribute values are visible to customers on the front-end.
 - :guilabel:`Hidden`: the attribute values are hidden from customers on the front-end.
 
+Lastly, in the optional :guilabel:`eCommerce Category` field, select a category from a drop-down
```

Comment:
Question: 
Are there any second-order consequences elsewhere in Odoo when setting this field's value? Or is this purely for the customer's benefit as a grouping tool on the Website front-end?

An example of what I mean would be when setting the main product's **Category**, this impacts a number of items in the Inventory app such as routing, removals/putaways, inventory valuation, etc.

Reply:
Great question! So, after some double-checking, it's my understanding that, since this is an 'eCommerce Category' (and not a 'Product Category'), this only affects the front-end/eCommerce/online store. So, I don't believe it has any second-order consequences. But totally valid (and smart) thing to consider. 👍